pkg/controller/networkpolicy/clusternetworkpolicy_test.go

```diff
@@ -896,6 +989,9 @@ func TestProcessClusterNetworkPolicy(t *testing.T) {
 			if tt.inputPolicy.Spec.Tier != "" {
 				c.tierStore.Add(&tierA)
 			}
+			if tt.inputPolicy.Spec.Egress != nil && tt.inputPolicy.Spec.Egress[0].ToService != nil {
+				c.endpointsStore.Add(&epA)
```

Comment:
why don't we add it unconditionally, would it be an issue?

Reply:
I think `epA` is not required in other cases. So I followed what we did on `tierA` above.

Reply:
Changed.